docs/guides/api-tokens.md

```diff
@@ -0,0 +1,88 @@
+# Protecting an API with Access Tokens
+
+Access Tokens can be used to authenticate users for your own site, or when allowing third-party developers to access your API. When making requests using access tokens, the token should be included in the `Authorization` header as a `Bearer` token.
+
+To issue tokens for users, the `UserModel` must use the `CodeIgniter\Shield\Authentication\Traits\HasAccessTokens` trait. The `UserModel` that ships with Shield already uses this trait.
+
+```php
+use CodeIgniter\Shield\Authentication\Traits\HasAccessTokens
+use CodeIgniter\Model;
+
+class UserModel extneds Model
+{
+    use HasAccessTokens;
+}
+```
+
+Tokens are issued with the `generateAccessToken()` method on the user. This returns a `CodeIgniter\Shield\Entities\AccessToken` instance. Tokens are hashed using a SHA-256 algorithm before being saved to the database. The access token returned when you generate it will include a `raw_token` field that contains the plain-text, un-hashed, token. You should display this to your user at once so they have a chance to copy it somewhere safe, as this is the only time this will be available. After this request, there is no way to get the raw token.
+
+The `generateAccessToken` method requires a name for the token. These are free strings and are often used to identify the user/device the token was generated from, like 'Johns MacBook Air'.
+
+```php
+$routes->get('/access/token', static function() {
+    $token = auth()->user()->generateAccessToken(request()->getVar('token_name));
+
+    return ['token' => $token->raw_token];
```

Comment:
CodeIgniter doesn't have fancy array response handling 😉 You've been hanging around "those other guys" too much. 
```suggestion
    return json_encode(['token' => $token->raw_token]);
```

Reply:
Lol I know. Darn day jobs :)